ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.tsx

```diff
@@ -249,5 +248,78 @@ export const MenuItem = (props: MenuItemProps) => {
   )
 }
 
+type MenutItemPropsDisabled = {
+  /** A value to represent the current Item */
+  value?: any
+  /**
+   * Children to display for menu item.
+   *
+   * @default value
+   */
+  children?: any
+  /** Optional styles for root element. */
+  style?: object
+  onClick?: any
+  selected?: any
+  active?: any
+  onHover?: any
+  disabled?: any
+  title?: any
+}
+
+const ItemRootDisabled = styled.option<{ active: boolean; disabled: boolean }>`
```

Comment:
Consider collapsing the `active` and `disabled` options into a single `active` option since `disabled` is the same as `!active`. This also prevents weird state issues like having both of them set to `true` or `false`